Erlang implementation of uuidv7

sg2342 opened the pull request.

sg2342 committed:
Erlang implementation of uuidv7

nalgeon commented:
Does not seem to work:

```text
main.erl:0: function main/1 undefined
escript: There were compilation errors.
```

https://onecompiler.com/erlang/42ghb2zxx

sg2342 committed:
appease https://onecompiler.com/erlang

- change main to take one (ignored) argument
- replace binary:encode_hex with something that works on OTP
  versions before 24

sg2342 commented:
it now works with the onecompiler.com erlang version: https://onecompiler.com/erlang/42ghcmets

nalgeon commented:
Thank you!
